### PR TITLE
add FixturesBundle & reload

### PR DESCRIPTION
**Pull request by shubaivan**
LGTM

### CODE REVIEW

**Comment by monobook** [comment]
Fixtures must be on Dev section of composer and app kernel

**Comment by monobook** [comment]
All dot folders must be ignored by global gitignore. Read this 
https://github.com/github/gitignore

**Comment by shubaivan** [comment]
what dot folders do you mean

**Comment by monobook** [comment]
.folder, look at the link

**Comment by shubaivan** [comment]
sorry, I mean, what dot folders should be added ?

**Comment by monobook** [comment]
.idea should be removed from gitignore, this gitignore for project. All other folders should be on global gitignore which defined by system. Once again look on repo, hint - find symfony and jetbrains and compare them...

**Comment by spolischook** [comment]
Don't commit file rights - http://stackoverflow.com/questions/1257592/how-do-i-remove-files-saying-old-mode-100755-new-mode-100644-from-unstaged-cha

File: bin/reload [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+require __DIR__ . '/base_script.php';
```
Comment (spolischook) [inline bin/reload:3]: so, where is it?

File: composer.json [hunk 2]
```diff
@@ -20,9 +20,11 @@
         "incenteev/composer-parameter-handler": "~2.0",
         "friendsofsymfony/rest-bundle": "~1.4",
         "jms/serializer-bundle": "0.13.*",
-        "nelmio/api-doc-bundle": "~2.7"
+        "nelmio/api-doc-bundle": "~2.7",
```
Comment (spolischook) [inline composer.json:23]: please, use home page as api documentation page

File: app/AppKernel.php [hunk 3]
```diff
@@ -20,10 +20,12 @@ public function registerBundles()
             new FOS\RestBundle\FOSRestBundle(),
             new JMS\SerializerBundle\JMSSerializerBundle(),
             new Nelmio\ApiDocBundle\NelmioApiDocBundle(),
-            new Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle()
+            new Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle(),
+
```
Comment (spolischook) [inline app/AppKernel.php]: empty blank not needed here

**Pull request merged**